## File "./lib/Maps.v", line ...:
## Warning: Notation app_ass is deprecated since 8.18. Use app_assoc instead.
## [deprecated-syntactic-definition-since-8.18,deprecated-since-8.18,deprecated-syntactic-definition,deprecated,default]

Index: lib/Maps.v
--- lib/Maps.v.orig
+++ lib/Maps.v
@@ -1129,16 +1129,16 @@ Module PTree <: TREE.
               match o with None => nil | Some x => (prev j, x) :: nil end ++
               l1);
       exists l2; split.
-      rewrite EQ, ! app_ass. auto.
-      rewrite EQ', ! app_ass. auto.
+      rewrite EQ, <- ! app_assoc. auto.
+      rewrite EQ', <- ! app_assoc. auto.
     + destruct (IHm i (xO j) H0) as (l1 & l2 & EQ & EQ').
       exists l1;
       exists (l2 ++
               match o with None => nil | Some x => (prev j, x) :: nil end ++
               xelements r (xI j));
       split.
-      rewrite EQ, ! app_ass. auto.
-      rewrite EQ', ! app_ass. auto.
+      rewrite EQ, <- ! app_assoc. auto.
+      rewrite EQ', <- ! app_assoc. auto.
     + subst o. exists (xelements l (xO j)); exists (xelements r (xI j)); split.
       rewrite prev_append_prev. auto.
       auto.
